build.sh

```diff
@@ -2,7 +2,7 @@
 
 # resolve python-version to use
 if [ "$PYTHON" == "" ] ; then
-    if ! PYTHON=$(command -v python || command -v python2 || command -v python 2.7)
+    if ! PYTHON=$(command -v python || command -v python2 || command -v python2.7)
```

Comment:
Should we look here in the same order as we do in the CMakeLists.txt above? I like the logic of starting with 2.7, then trying 2 and finally just python.